### PR TITLE
Slight update for accessing extension dev tools, as well as troublesh…

### PR DESCRIPTION
**Pull request by DaedalusG**
…ooting for a Safari bug

### CODE REVIEW

**Review by virginiaulrich — APPROVED** [review]
Would we also include Firefox?

File: doc/integration/browser_extension.md [hunk 1]
```diff
@@ -136,12 +136,24 @@ In rare cases, Chrome can get into the state where the option to **Enable Source
...
 ![Chrome extension console and network activity](https://sourcegraphstatic.com/ChromeExtensionConsoleNetworkActivity.gif)
 
+1) Right click the sourcegraph browser extension icon
+2) Select Manage Extensions
+3) Under Inspect Views select background page, this will open a dev console to the extension background page
+4) In the developer console select the network tab
+
+In Safari:
+
+1) Ensure you have access to the develop tab by enterining Safari > Preferences, at the bottom of the preference UI check the box labelled `Show Develop menu in menu bar`
```
[the remark this answers is not in the document]
Comment (muratsu) [reply doc/integration/browser_extension.md:153]: Typo `enterining`

File: doc/integration/browser_extension.md [hunk 2]
```diff
@@ -150,6 +162,12 @@ Since `v3.14.0+`, the Sourcegraph browser extension can only authenticate with S
 
 Previously, the Sourcegraph browser extension was able to authenticate with instances that hadn't enabled tls / ssl. However, modern web browsers have started to adopt and implement [an IETF proposal](https://web.dev/samesite-cookies-explained/) that removes the deprecated logic that allowed this behavior. Please configure [HTTPS](../admin/tls_ssl.md) in order to continue using the browser extension with your private instance.
 
+### `The string did not match the expected pattern` error in Safari
+
+The above validation error occurs in safari when the browser when trys to parse a non-json text as JSON. Communication between the graphql server & browser extension is always assumed to be JSON. However when a proxy/saml is used with Sourcegraph, sometimes text/html can occur in network communication.
```
Comment (muratsu) [inline doc/integration/browser_extension.md:167]: The above validation error occurs in Safari when the browser tries to parse a non-JSON text as JSON. Communication between the graphql server & browser extension is always assumed to be JSON. However, when a proxy/saml is used with Sourcegraph, sometimes the proxy can send text/html instead.

**Review by muratsu — APPROVED** [review]
Thanks for writing this down!

File: doc/integration/browser_extension.md (hunk 2 above)
Comment (DaedalusG) [reply doc/integration/browser_extension.md:167]: ✅

**Pull request merged**